### PR TITLE
Fix type error on php7.4

### PR DESCRIPTION
**Pull request by alexander-schranz**
In some cases it seems that a node path is a integer.

### CODE REVIEW

**Comment by dbu** [comment]
i guess the stupid behaviour of php to convert array keys to int when they look like int...

the fix seems right to me. to make the tests green on php 7.4, we would need to upgrade phpunit :-| 
can you please try to do that? i think we should make this a new minor version and drop php 5.6/7.0 support to not overcomplicate this.

please add notes to the CHANGELOG.md file too.

**Comment by alexander-schranz** [comment]
@dbu not sure what I did wrong that travis is not longer trigger 🙈 maybe it wants to keep php 5.5 and 7.0 😄

File: tests/Jackalope/ObjectManagerTest.php [hunk 1]
```diff
@@ -55,12 +55,11 @@ public function testRegisterUuid()
...
     }
 
-    /**
-     * @expectedException \RuntimeException
-     * @expectedExceptionMessage Object path for UUID "1234" has already been registered to "/path/to/this"
-     */
     public function testRegisterUuidAlreadyMapped()
     {
+        $this->expectException(\RuntimeException::class);
+        $this->expectExceptionMessage('Object path for UUID "1234" has already been registered to "/path/to/this"');
```
Comment (dbu) [inline tests/Jackalope/ObjectManagerTest.php]: please put this after the first `registerUuid` call. the upside of having the expect exceptions in code is that they can be at the precisely correct place.

Comment (alexander-schranz) [reply tests/Jackalope/ObjectManagerTest.php]: done

**Comment by dbu** [comment]
> could you please add something to the changelog file?

the changelog is still missing, otherwise looks good to merge to me.

**Comment by alexander-schranz** [comment]
@dbu ups forget to push is now there.

**Comment by dbu** [comment]
thanks!

**Comment by dbu** [comment]
and tagged https://github.com/jackalope/jackalope/releases/tag/1.4.0

**Pull request merged**